great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -892,6 +990,7 @@ def _apply_runtime_environment(
                         )
                         setattr(rule.domain_builder, property_key.value, property_value)
                     except AttributeError:
+                        # Skip every directive that is not defined property of "DomainBuilder" object of "domain_type".
                         pass
```

Comment:
Could we use a `log.debug()` statement here?

Reply:
@cdkini I believe that this message will not be valuable.  This is one of those very rare cases when the exception handling has to help implement the logic (i.e., this is a normal case, for which we do not want warnings in notebooks).